Header header
float32 weightA
float32 distance
float32 humidity
bool vibrations
float32 weightB
float32 potassium
float32 nitrogen
float32 phosphorus
float32 ph